mmv1/products/vertexai/api.yaml

```diff
@@ -997,3 +999,208 @@ objects:
           The update method to use with this Index. The value must be the followings. If not set, BATCH_UPDATE will be used by default.
           * BATCH_UPDATE: user can call indexes.patch with files on Cloud Storage of datapoints to update.
           * STREAM_UPDATE: user can call indexes.upsertDatapoints/DeleteDatapoints to update the Index and the updates will be applied in corresponding DeployedIndexes in nearly real-time.
+
+# Vertex AI Models
+  - !ruby/object:Api::Resource
+    name: Model
+    base_url: projects/{{project}}/locations/{{location}}/models
+    create_url: projects/{{project}}/locations/{{location}}/models:upload
+    self_link: 'projects/{{project}}/locations/{{location}}/models/{{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/vertex-ai/docs'
+      api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.models'
+    description: "A trained machine learning Model."
```

Comment:
`parentModel` is a supported field in the API, allowing you to update a preexisting model. Given we don't allow that to be specified, does that mean our resource can't add a new version to a model? (it's fine if we can't- I just want to make sure my understanding is correct)

What happens if you add two resources with the same `name`/`model_id` in the same configuration? Does one return an error?

Reply:
This would result in a 409 error from the api.